cli/src/pcluster/resources/imagebuilder/parallelcluster_validate.yaml

```diff
@@ -235,9 +235,8 @@ phases:
                   rpm -qa | grep libfabric && rpm -qa | grep efa-
                   [[ $? -ne 0 ]] && echo "Check efa rpm failed" && exit 1
 
-                  echo "Checking intel mpi installed..."
-                  rpm -qa | grep intel-mpi
-                  [[ $? -ne 0 ]] && echo "Check intel-mpi rpm failed" && exit 1
+                  echo "Checking Intel MPI installed..."
+                  [[ ! -d "/opt/intel/mpi" ]] && echo "Check Intel MPI failed" && exit 1
```

Comment:
Is it possible to verify the installation in a more robust way rather than a directory check?
For instance calling the command version or similar.

Reply:
Done, I'm not checking the specific 2021 version to avoid the need to modify cli package when upgrading the package from the cookbook.

We already have a check for the version in kitchen tests.